test/new-e2e/tests/fips-compliance/fips_nix_test.go

```diff
@@ -68,12 +68,8 @@ instances: [{}]
 }
 
 func (v *LinuxFIPSComplianceSuite) TestFIPSDefaultConfig() {
-	_, err := v.Env().RemoteHost.Execute("sudo GOFIPS=0 datadog-agent status")
-	require.NotNil(v.T(), err)
-	assert.Contains(v.T(), err.Error(), "the 'requirefips' build tag is enabled, but it conflicts with the environment variable GOFIPS=0 which would disable FIPS mode")
```

Comment:
`GOFIPS=0` is a noop since https://github.com/microsoft/go/pull/1708, so just removing this test